gems/pending/middleware/events/middleware_event_monitor.rb

```diff
@@ -0,0 +1,31 @@
+class MiddlewareEventMonitor
+  def initialize(ems, start_time)
+    @ems = ems
+    @alerts_client = ems.alerts_connect
+    # TODO: can we store and resume where we left off?  To avoid duplicates, only look for events now or later,
+    # so basically we miss events when we're not running...
+    @start_time = start_time || Time.now.to_i * 1000
```

Comment:
if you have a method signature like
```ruby
def initialize(ems, start_time = Time.now.to_i * 1000)
```
then its really an optional argument. Now you'd have to pass `nil` to satisfy the interpreter